### PR TITLE
Change Bluemix to IBM Cloud

### PR DESCRIPTION
**Pull request by dolph**

### CODE REVIEW

File: ?
Comment (markstur) [inline ?:1]: "IBM Cloud Cloudant" doesn't sound right.  Could be just Cloundant or IBM Cloundant. The longer name in docs is "IBM® Cloudant® NoSQL DB for IBM Cloud", but usually it is shortened.

Comment (markstur) [inline ?:1]: this would break the link from steps if it wasn't already broken by the translation

Comment (markstur) [inline ?:1]: this would break the link from steps if it wasn't already broken by the translation

Comment (markstur) [inline ?:1]: Cloud Cloudant again (doesn't look bad here, just sounds funny)

Comment (markstur) [inline ?:1]: breaks the link from above

**Review by markstur — CHANGES_REQUESTED** [review]

**Comment by stevemar** [comment]
this was fixed in the main README

**Pull request closed without merging**